src/colorizer/recorders/Mp4VideoRecorder.ts

```diff
@@ -83,6 +83,7 @@ export default class Mp4VideoRecorder extends CanvasRecorder {
               width: width,
               height: height,
             },
+            fastStart: "in-memory",
```

Comment:
Needed to specify this with the new version of `mp4-muxer`. More documentation on this here: https://github.com/Vanilagy/mp4-muxer?tab=readme-ov-file#faststart-required